homeassistant/components/switchbee/__init__.py

```diff
@@ -22,7 +22,6 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     central_unit = entry.data[CONF_HOST]
     user = entry.data[CONF_USERNAME]
     password = entry.data[CONF_PASSWORD]
-
```

Comment:
Side note: I think you should keep the carriage return here - it makes it easier to read with the carriage return kept in place

Reply:
how? can you suggest?

Reply:
Suggestions don't work on "deleted" lines. You'll have to edit the file

Reply:
I know, I am just looking up carriage return and I understand its the special character '\r' which return the console cursor to the beginning of the line, I don't understand how it fits here...